src/Analysis/Codelyzer.Analysis.CSharp/Handlers/ClassDeclarationHandler.cs

```diff
@@ -21,6 +20,7 @@ public class ClassDeclarationHandler : UstNodeHandler
             if (classSymbol != null && classSymbol.BaseType != null)
             {
                 ClassDeclaration.BaseType = classSymbol.BaseType.ToString();
+                ClassDeclaration.BaseTypeOriginalDefinition = classSymbol.BaseType.OriginalDefinition.ToString();
```

Comment:
Write a wrapper method to get the original definition. Please check for nulls inside.

Reply:
Done